src/formatDate.js

```diff
@@ -50,7 +50,56 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let day, month, year;
```

Comment:
try to use object for that

Reply:
the object is helpful if, for example, you need to use the property 'YYYY' and your object doesn't have any, you already knew that you need to transform value from 'YY' property.